packages/components/app/styles/components/icon-tile.scss

```diff
@@ -58,13 +57,13 @@ $size-props: (
     'extra-icon-size': 0.75rem,
   ),
   "medium": (
-      "size": 2.5rem, // 40px
-      "border-radius": 6px,
-      "icon-size": 1.5rem, // 24px
-      "logo-size": 1.75rem, // 28px
-      'extra-size': 1.5rem, // 24px
-      'extra-border-radius': 5px,
-      'extra-icon-size': 1rem,
+    "size": 2.5rem, // 40px
+    "border-radius": 6px,
+    "icon-size": 1.5rem, // 24px
+    "logo-size": 1.75rem, // 28px
+    'extra-size': 1.5rem, // 24px
+    'extra-border-radius': 5px,
+    'extra-icon-size': 1rem,
```

Comment:
I'd like to see a consistent quote style. 

(My personal preference has been to use double quotes within HTML & CSS and single quotes within JS.)

Reply:
Yes, this will be discussed in https://github.com/hashicorp/design-system/pull/543